src/hotspot/share/runtime/continuationFreezeThaw.cpp

```diff
@@ -389,14 +389,14 @@ class FreezeBase : public StackObj {
   // slow path
   frame freeze_start_frame();
   frame freeze_start_frame_safepoint_stub(frame f);
-  NOINLINE freeze_result freeze(frame& f, frame& caller, int callee_argsize, bool callee_interpreted, bool top);
+  NOINLINE freeze_result recurse_freeze(frame& f, frame& caller, int callee_argsize, bool callee_interpreted, bool top);
```

Comment:
But now there are two functions named recurse_freeze aren't there?  Same with recurse_thaw?

Reply:
Never mind. I thought I saw an extra recurse_freeze.  The name recurse_thaw is better than thaw_one_frame.

Reply:
There is only one recurse_freeze. But there are a couple of methods with the recurse_ prefix because they are all called recursively during freeze/thaw. freeze() and thaw() were missing that prefix.